src/phpDocumentor/Configuration/Definition/Version2.php

```diff
@@ -152,7 +152,12 @@ public function getConfigTreeBuilder() : TreeBuilder
      *
      * @param array<string, string|array<string, mixed>> $values
      *
-     * @return array<string, array<int|string, mixed>|string>
+     * @return array{
+     * configVersion: string, title: array<string, mixed>|string, paths: array{output: mixed, cache: mixed},
+     * version: array{0: array{number: string, api: array{0: array{default-package-name: mixed,
+     * source: array{paths: array<string>}, ignore: array{paths: array<string>}, extensions: array{extensions: mixed},
+     * visibilities: non-empty-list<string>|null, markers: array{markers: mixed}}}}}, templates: array
```

Comment:
`markers: mixed` should be `markers: non-empty-list<string>` 
`templates: non-empty-list<string>`